app/controllers/CreateSubscriptionAndUpdateEnrolment.scala

```diff
@@ -68,6 +68,7 @@ trait CreateSubscriptionAndUpdateEnrolment extends Logging {
           } else {
             Future.successful(Redirect(routes.PreRegisteredController.onPageLoad(false)))
           }
+        case _ => Future.successful(Redirect(routes.PreRegisteredController.onPageLoad(false)))
```

Comment:
this should be `Future.successful(Redirect(routes.ThereIsAProblemController.onPageLoad()))` because we don't know the error here but it won't be that they are already registered

Reply:
changes done as per comments